scapy/contrib/vqp.py

```diff
@@ -52,15 +52,13 @@ class VQPEntry(Packet):
             3080: "ResMACAddress"
         }),
         FieldLenField("len", None),
-        ConditionalField(IPField("datatom", "0.0.0.0"),
-                         lambda p: p.datatype == 3073),
-        ConditionalField(MACField("data", "00:00:00:00:00:00"),
-                         lambda p: p.datatype == 3078),
-        ConditionalField(MACField("data", "00:00:00:00:00:00"),
-                         lambda p: p.datatype == 3080),
-        ConditionalField(StrLenField("data", None,
-                                     length_from=lambda p: p.len),
-                         lambda p: p.datatype not in [3073, 3078, 3080]),
+        MultipleTypeField([
+            (IPField("datatom", "0.0.0.0"), lambda p: p.datatype == 3073),
+            (MACField("data", "00:00:00:00:00:00"),
```

Comment:
This is wrong. A multipletypefield should has all its fields with the same name.